### PR TITLE
7.0 backport report custom filename

### PR DESCRIPTION
**Pull request by sebastienbeau**
Just backport the module, really usefull ! Thanks Therp team ;)

### CODE REVIEW

File: report_custom_filename/README.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Custom report filenames
```
Comment (yvaucher) [inline report_custom_filename/README.rst]: AFAIK in version 7.0 description needs no be in `__openerp__.py`

Comment (sebastienbeau) [inline report_custom_filename/README.rst]: Thanks Yannick, I also fix the icon path

**Comment by pedrobaeza** [comment]
Thanks! :+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by florian-dacosta** [comment]
:+1:

**Pull request merged**